app/helpers/application_helper.rb

```diff
@@ -40,7 +40,11 @@ def phase_label(phase)
   end
 
   def formatted_date(date)
-    Date.parse(date).strftime('%d/%m/%Y')
+    Date.parse(date).strftime('%d/%m/%Y') # eg 03/08/2015
+  end
+
+  def human_readable_date(date)
+    Date.parse(date).strftime('%e %B %Y') # eg 3 August 2015
```

Comment:
Another way of doing this is to use a relative date like "over 3 years ago". I think this is the rails method for doing that: https://apidock.com/rails/ActionView/Helpers/DateHelper/time_ago_in_words

I don't know which is more useful in our case though.

Reply:
The '3 August 2015' format was what was requested - but that `time_ago_in_words` is very neat indeed! A lot of very nice helpers in Ruby.